controllers/infra_health_dashboard.json

```diff
@@ -1,641 +1,1083 @@
-                    "thresholds":[
                     ],
-                    "timeFrom":null,
-                    "timeShift":null,
-                    "title":"Rates",
-                    "tooltip":{
-                        "shared":true,
-                        "sort":0,
-                        "value_type":"individual"
+                    "thresholds": [],
+                    "timeFrom": null,
+                    "timeShift": null,
+                    "title": "Rates",
+                    "tooltip": {
+                        "shared": true,
+                        "sort": 0,
+                        "value_type": "individual"
                     },
-                    "type":"graph",
-                    "xaxis":{
-                        "buckets":null,
-                        "mode":"time",
-                        "name":null,
-                        "show":true,
-                        "values":[
-                        ]
+                    "type": "graph",
+                    "xaxis": {
+                        "buckets": null,
+                        "mode": "time",
+                        "name": null,
+                        "show": true,
+                        "values": []
                     },
-                    "yaxes":[
+                    "yaxes": [
                         {
-                            "format":"short",
-                            "label":null,
-                            "logBase":1,
-                            "max":null,
-                            "min":null,
-                            "show":true
+                            "format": "short",
+                            "label": null,
+                            "logBase": 1,
+                            "max": null,
+                            "min": null,
+                            "show": true
                         },
                         {
-                            "format":"short",
-                            "label":null,
-                            "logBase":1,
-                            "max":null,
-                            "min":null,
-                            "show":true
+                            "format": "short",
+                            "label": null,
+                            "logBase": 1,
+                            "max": null,
+                            "min": null,
+                            "show": true
                         }
                     ]
                 }
             ],
-            "repeat":null,
-            "repeatIteration":null,
-            "repeatRowId":null,
-            "showTitle":true,
-            "title":"Flows",
-            "titleSize":"h6"
+            "repeat": null,
+            "repeatIteration": null,
+            "repeatRowId": null,
+            "showTitle": true,
+            "tags": [
+                "flows"
+            ],
+            "title": "Flows",
+            "titleSize": "h6"
+        },
+        {
```

Comment:
If you don't mind me hijacking your PR :-)
I would very well see this kind of addition:
(this is a query that shows the "flows overhead" generated by netobserv itself compared with the total cluster flows. In other words, a way to see if netobserv is triggering a "runaway reaction" as we've already seen in the past, e.g. when we set too small batch sizes.


```suggestion
        },
        {
            "collapse": false,
            "height": "250px",
            "panels": [
                {
                    "aliasColors": {},
                    "bars": false,
                    "dashLength": 10,
                    "dashes": false,
                    "datasource": "prometheus",
                    "fill": 1,
                    "fillGradient": 0,
                    "gridPos": {
                        "h": 20,
                        "w": 25,
                        "x": 0,
                        "y": 0
                    },
                    "id": 3,
                    "legend": {
                        "alignAsTable": false,
                        "avg": false,
                        "current": false,
                        "max": false,
                        "min": false,
                        "rightSide": false,
                        "show": true,
                        "sideWidth": null,
                        "total": false,
                        "values": false
                    },
                    "lines": true,
                    "linewidth": 1,
                    "links": [],
                    "nullPointMode": "null",
                    "percentage": false,
                    "pointradius": 5,
                    "points": false,
                    "renderer": "flot",
                    "repeat": null,
                    "seriesOverrides": [],
                    "spaceLength": 10,
                    "stack": false,
                    "steppedLine": false,
                    "targets": [
                        {
                            "expr": "100 * sum(rate(netobserv_namespace_flows_total{SrcK8S_Namespace='netobserv'}[1m]) or rate(netobserv_namespace_flows_total{SrcK8S_Namespace!='netobserv',DstK8S_Namespace=~'netobserv'}[1m])) / sum(rate(netobserv_namespace_flows_total[1m]))",
                            "format": "time_series",
                            "intervalFactor": 2,
                            "legendFormat": "% overhead",
                            "refId": "A"
                        }
                    ],
                    "thresholds": [],
                    "timeFrom": null,
                    "timeShift": null,
                    "title": "Percentage of flows generated by NetObserv own traffic",
                    "tooltip": {
                        "shared": true,
                        "sort": 0,
                        "value_type": "individual"
                    },
                    "type": "graph",
                    "xaxis": {
                        "buckets": null,
                        "mode": "time",
                        "name": null,
                        "show": true,
                        "values": []
                    },
                    "yaxes": [
                        {
                            "format": "short",
                            "label": null,
                            "logBase": 1,
                            "max": null,
                            "min": null,
                            "show": true
                        },
                        {
                            "format": "short",
                            "label": null,
                            "logBase": 1,
                            "max": null,
                            "min": null,
                            "show": true
                        }
                    ]
                }
            ],
            "repeat": null,
            "repeatIteration": null,
            "repeatRowId": null,
            "showTitle": true,
            "tags": [
                "flows",
                "overhead",
                "flows-overhead"
            ],
            "title": "Flows Overhead",
            "titleSize": "h6"
        },
        {
```

Of course I can also open a separate PR

Reply:
Let's have it ! Thanks